src/puppetlabs/puppetdb/query_eng.clj

```diff
@@ -133,6 +133,17 @@
              (query->sql remaining-query entity version paging-options)]
          (jdbc/with-query-results-cursor results-query (comp row-fn munge-fn)))))))
 
+;; Do we still need this, i.e. do we need the pass-through, and the
+;; strict selectivity in the caller below?
```

Comment:
Yeah, though with some luck we eventually might not care about the parse-string functions at all.